thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/v2/AlertTaskRunnerV2.java

```diff
@@ -58,15 +63,18 @@
   private final MergedAnomalyResultManager anomalyMergedResultDAO;
   private final AlertConfigManager alertConfigDAO;
   private final MetricConfigManager metricConfigManager;
+  private final GroupedAnomalyResultsManager groupedAnomalyResultsDAO;
 
   private AlertConfigDTO alertConfig;
   private ThirdEyeAnomalyConfiguration thirdeyeConfig;
   private AlertFilterFactory alertFilterFactory;
+  private final long DEFAULT_MAX_ALLOWED_MERGE_GAP = 14400000L;
```

Comment:
This is used for time-based merge of grouped anomalies. We could make it configurable when this functionality is used more often.